src/components/media-player/ha-media-player-browse.ts

```diff
@@ -92,11 +94,54 @@ export class HaMediaPlayerBrowse extends LitElement {
     this._navigate(item);
   }
 
+  private _renderError(err: { message: string; code: string }) {
+    if (err.message === "Path does not exist.") {
+      return html`
+        <h2>No local media found.</h2>
+        <p>
+          It looks like you have not yet created a media directory.
+          <br />Create a directory with the name <b>"media"</b> in the
+          configuration directory of Home Assistant (${this.hass.config.config_dir}). <br />Place your video,
+          audio and image files in this directory to be able to browse and play
+          them in the browser or on supported media players.
+        </p>
+
+        <p>
+          Check the
+          <a
+            href="https://www.home-assistant.io/integrations/media_source/#local-media"
+            target="_blank"
+            rel="noreferrer"
+            >documentation</a
+          >
+          for more info
+        </p>
+      `;
+    }
+    return err.message;
+  }
+
   protected render(): TemplateResult {
     if (this._loading) {
       return html`<ha-circular-progress active></ha-circular-progress>`;
     }
 
+    if (this._error && !this._mediaPlayerItems.length) {
+      if (this.dialog) {
+        this._closeDialogAction();
+        showAlertDialog(this, {
+          title: this.hass.localize(
+            "ui.components.media-browser.media_browsing_error"
+          ),
+          text: this._renderError(this._error),
+        });
+      } else {
+        return html`<div class="container error">
+          ${this._renderError(this._error)}
+        </div>`;
+      }
+    }
```

Comment:
What if they run into an error that is a file existed when the refresh the browse. But since has been deleted. They would try to click the item and would get path isn't found error. Showing that in children container seems weird. We should still use the dialog and key their place in the media browser